src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    protected String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    abstract double getArea();

```

Comment:
make it and all overriding methods public, or create interface AreaCalcultor, same as drawable